vm-setup/roles/v1aX_integration_test/tasks/remediation.yml

```diff
@@ -3,6 +3,9 @@
     set_fact:
       NUMBER_OF_BMH: "{{ NUM_OF_MASTER_REPLICAS|int +  NUM_OF_WORKER_REPLICAS|int }}"
 
+  - name: Obtain target cluster kubeconfig
+    shell: "kubectl get secrets {{ CLUSTER_NAME }}-kubeconfig -n {{ NAMESPACE }} -o json | jq -r '.data.value'| base64 -d > /tmp/target.yaml"
```

Comment:
Can you please put the target cluster name in the created kubeconfig file name ?

Reply:
Done.